lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Yes, `gollum` supports the use of bare repositories. This is essential for use cases where users can also push to the repo over `http`, for example.